### DISCUSSION TITLE
Is there some sort of repeater field for custom attributes?

### DISCUSSION THREAD

**Discussion by JordanBettridge** [saleor/saleor#10022] [Q&A] [answered]
Hi guys, currently looking into using Saleor as an alternative CMS and e-commerce platform to Wordpress. We regularly use the ACF plugin on Wordpress which allows us to add custom fields/attributes to page types, etc. but the only issue is, I don't see some sort of repeatable field here on Saloer? I saw you can do booleans, text, etc. but I don't see repeater fields, for example say we have some USPs displayed as icons, and sometimes I want to add 5 icons, sometimes I want to add 10, I cant see a repeatable field? One thing I also don't see is adding a WYSISYG editor as an attribute?

**Comment by karolkielecki** [comment]
Hi @Jordz2203! Saleor currently supports basic CMS functionality. We'd like to support such features (mentioned by you) in the future, but there is no ETA. For now, I can only suggest you check if you can create an integration (an app) that will support your use case. https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts

**Comment by timuric** [comment, marked as answer]
@Jordz2203 hopefully we would start working on it in August 2022, it is a highly requested feature and we would like to add more advance CMS features as well.

I have added item to the roadmap https://github.com/orgs/saleor/projects/3/views/1

**Reply by JordanBettridge** [reply]
Hi Timur, thats awesome. Super happy to hear it! Ive been really juggling between Saloer and some other CMS systems, the reason I love Saleor is its product/commerce centric. I just really wish the CMS side was a little more advance, you guys already essentially support custom page/post types and attributes. Im sure you guys would do much better if you supported repeatable fields and maybe upgraded the WYSIWYG editor a little more, purely because I know of 2 companies that didn't make the switch to Saleor because those 2 functions were lacking. 

Super excited to see the changes coming!
Love From South Africa,
Jordan

**Reply by timuric** [reply]
@Jordz2203 as a workaround (not for all cases perhaps) meantime you can create pages (which are basically blocks) and create references to them from the product. This way you could create for example blocks with recommendation content, like images, text, etc, and assign as many as you want to products.

**Reply by JordanBettridge** [reply]
@timuric That actually works great thank you. I haven't taken a look at your backend architecture but from my own experience. I don't think it would be a difficult take to do this.

On the left-hand main navigation, you could make a section where there is a page, and on the page put all the page types (maybe in a grid format). Then upon clicking each page type, you ONLY see the pages of this type, whilst this is accomplishable by just going to the pages section, I feel from a non-developer perspective it would make the page type user experience more friendly.

So keep the pages URL /dashboard/pages/

But also add an area that has a list of page types, maybe in a grid format, so all the page types can be easily navigated to

**Reply by timuric** [reply]
@Jordz2203 This is definitely going to be improved once we start CMS-related sprints, for now, you can use Filter to browse specific types.

**Reply by JordanBettridge** [reply]
Hi @timuric, following up here to see if theres been any progress? Been almost 2 years and I was really hoping there has been!
